pkg/logql/metrics.go

```diff
@@ -314,23 +334,36 @@ func RecordVolumeQueryMetrics(
 		latencyType = latencyTypeSlow
 	}
 
+	rangeType := "range"
+	if step == 0 {
+		rangeType = "instant"
+	}
+
 	level.Info(logger).Log(
 		"latency", latencyType,
 		"query_type", queryType,
 		"query", query,
 		"query_hash", HashedQuery(query),
+		"start", start.Format(time.RFC3339Nano),
+		"end", end.Format(time.RFC3339Nano),
+		"start_delta", time.Since(start),
+		"end_delta", time.Since(end),
+		"range_type", rangeType,
+		"step", step,
+		"limit", limit,
 		"length", end.Sub(start),
 		"duration", time.Duration(int64(stats.Summary.ExecTime*float64(time.Second))),
 		"status", status,
 		"splits", stats.Summary.Splits,
 		"total_entries", stats.Summary.TotalEntriesReturned,
-		"cache_volume_results_req", stats.Caches.VolumeResult.EntriesRequested,
-		"cache_volume_results_hit", stats.Caches.VolumeResult.EntriesFound,
-		"cache_volume_results_download_time", stats.Caches.VolumeResult.CacheDownloadTime(),
```

Comment:
if this is the volume metrics endpoint, woudln't we want the cache stats included with it?

Reply:
I don't understand the question; those stats are still included

Reply:
ooohhh, I was just looking at the names. not the values setting them.

I think I would prefer to leave "volume" in the name?

hrmmm still thinking about this

Reply:
I removed them because they felt unnecessary (since it's a volume query after all) and it made the keys very long

Reply:
my argument towards keeping volume in the name would be to avoid ambiguity and removing a case where I could see a common mistake made:

```
sum(sum_over_time({} |= "metrics.go | logfmt | unwrap cache_results_hit[1m]))
/
sum(sum_over_time({} |= "metrics.go | logfmt | unwrap cache_results_req[1m]))
```

This is a query I've done a lot in the past to see how effective our results cache is, but with this change I would both need to know and remember it would be including stats for volume queries which I think is too error prone. WDYT?

Reply:
Fair point 👍 I'll add them back in